bitbots_vision/scripts/bitbots_imageloader.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+import argparse
+import cv2
+import os
+import sys
+import rospy
+# Check if progressbar is installed
+try:
+    import progressbar
+    progressbar_installed = True
+except ImportError:
+    progressbar_installed = False
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge, CvBridgeError
+
+
+class LoadImages:
+    def __init__(self, path, frame="/camera_link", topic="image_raw", fps=10, loop=False, pgbar=False):
```

Comment:
I think that camera_optical_frame is better here.

Reply:
Yes, I agree with you.